rendering.js

```diff
@@ -64,14 +64,16 @@ function renderLivesRemaining({ livesRemaining }) {
 }
 
 function renderFallingObjects({ fallingObjects }) {
-    const fallingObjectsContainer = document.getElementById(
-        EL_IDS.fallingObjectsList,
+    const fallingObjectColumnEls = document.querySelectorAll(
```

Comment:
Couldn't you do the old `getElementById` call to get the outer `<ul>` then use the `children` property to get a collection of `fallingObjectColumnEls`, and still access them the same way?

It should be the same result, but I think using `getElementById` is going to be much faster than `querySelectorAll` (near constant vs linear time DOM query) - though I'm not actually sure, more just guessing

Reply:
I think you're right, query Id is much faster since it can stop the search once it finds the element.